core/src/player.rs

```diff
@@ -965,6 +969,11 @@ impl Player {
                                 false.into(),                            /* cancelable */
                                 key_char.map_or(0, |c| c as u32).into(), /* charCode */
                                 (key_code as u32).into(),                /* keyCode */
+                                0.into(),                                /* keyLocationValue */
```

Comment:
Can you add a FIXME comment here?

Reply:
Added a TODO for that, for commandKey, and for refining ctrlKey.